docs/installation/index.rst

```diff
@@ -1,10 +1,13 @@
-Installation Guide
-==================
+Installation
+============
 
 .. toctree::
    :maxdepth: 2
-   
+
    instructions
    configuration-files
    distributed-installation
    migration
```

Comment:
Reminder:  no content yet for Distributed Installation nor for Migration from Pulp 2.

You mentioned skipping Plugins for now, but then following that section, there is still content missing from Workflows and Use Cases, Release Notes, and CLI User Guide.